### PR TITLE
send invitation

### PR DESCRIPTION
**Pull request by nicolasparada**
Adding the following command:

```bash
calyptia create invitation user@example.org
```

### CODE REVIEW

**Review by pwhelan — DISMISSED** [review]
LGTM

**Comment by nicolasparada** [comment]
Waiting on https://github.com/calyptia/cloud-e2e/pull/79

**Review by niedbalski — APPROVED** [review]
LGTM

File: cmd/invitation/send_invitation.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+				cmd.Printf("invitation sent to %q successfully\n", email)
+			}
+
+			return nil
+		},
+	}
+
+	fs := cmd.Flags()
+	fs.StringVar(&redirectURI, "redirect-uri", redirectURI, "Redirect URI for the invitation, leave the default value if you don't know what it is")
```
Comment (niedbalski) [inline cmd/invitation/send_invitation.go:42]: Where are you using this value? You should set it to "https://core.calyptia.com" and use redirectURI

**Review by niedbalski — CHANGES_REQUESTED** [review]
Minor nitpicks

**Comment by nicolasparada** [comment]
@niedbalski I think the change to the redirect url won't pick the correct default value. It will always use the prod ui.

**Pull request merged**